config/docker/nginx-cors-permissive.conf

```diff
@@ -118,11 +101,16 @@ server {
     proxy_set_header X-Forwarded-For $remote_addr;
   }
 
+
+# Only needed if you user the control of the IP address by user
```

Comment:
Only needed if you user the control of the IP address by user => Only needed if the control of the user IP address is activated